go.mod

```diff
@@ -430,3 +430,5 @@ replace (
 
 // Exclude this version because the version has an invalid checksum.
 exclude github.com/docker/distribution v2.8.0+incompatible
+
+replace github.com/elastic/elastic-agent-libs => github.com/belimawr/elastic-agent-libs v0.2.9-0.20240411111018-dc1fe80ad648
```

Comment:
should this replace be removed?

Reply:
Yes, as soon as https://github.com/elastic/elastic-agent-libs/pull/171 gets merged.